packages/generator-one-app-module/generators/app/templates/root-module/__tests__/appConfig.spec.js

```diff
@@ -0,0 +1,45 @@
+import AppConfig from '../src/appConfig';
+
+describe('appConfig', () => {
+  it('should contain accurate csp', () => {
+    expect(AppConfig.csp).toBeDefined();
+    expect(typeof AppConfig.csp).toBe('string');
+  });
+
+  describe('csp', () => {
+    beforeEach(() => {
+      jest.resetModules();
+      process.env.ONE_CLIENT_REPORTING_URL = 'example.com';
+    });
+
+    it('should take the HTTP_ONE_APP_DEV_CDN_PORT environment variable into account for local development csp additions', () => {
+      process.env.NODE_ENV = 'development';
+      process.env.HTTP_ONE_APP_DEV_CDN_PORT = 5000;
+      // eslint-disable-next-line global-require
+      const cspString = require('../src/csp').default;
+      const sanitizedCspString = cspString.replace(/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}:5000/g, '0.0.0.0:5000');
```

Comment:
Can you add a code comment above this line explaining what this regex is doing and why?

Reply:
fixed, added
`// replaces local ip with 0.0.0.0 for consistent snapshots`